src/cascadia/CascadiaPackage/CascadiaPackage.wapproj

```diff
@@ -142,29 +131,20 @@
     </ItemGroup>
   </Target>
   <!-- End workaround -->
-
   <ItemGroup>
     <ProjectReference Include="..\WindowsTerminal\WindowsTerminal.vcxproj" />
     <ProjectReference Include="..\..\host\exe\Host.EXE.vcxproj" />
   </ItemGroup>
-
   <!-- Stomp the path to conhost.exe.
     This task will copy OpenConsole.exe to the appx as conhost.exe, and place it
     adjacent to WindowsTerminal.exe.
    -->
   <Target Name="OpenConsoleStompSourceProjectForWapProject" BeforeTargets="_ConvertItems">
-    <ItemGroup>
-      <!-- Stomp all "SourceProject" values for all incoming dependencies to flatten the package. -->
-      <_TemporaryFilteredWapProjOutput
-        Include="@(_FilteredNonWapProjProjectOutput)" />
+    <ItemGroup><!-- Stomp all "SourceProject" values for all incoming dependencies to flatten the package. -->

```

Comment:
i guess while you're changing this file, these are actually spaces that are encoded wrong (!) and all need to be fixed

Reply:
I think I got it.